armada-ths/ais
# Bugfix broken save

A handful of companies had the issue that the save or submit button did not work. Finally figured out that one reason causing this is if they have filled in the form the wrong way (letters in int fields in student matching) or if they have left some int fields blank (products).

I have:
- Added help texts to student matching int fields, packages to/from fair, and eletric outlets
- Set all product int field to not required
- Fixed so the form handles null for product int field 

To test:
1. Check if help texts show up/look good
2. Try to save form with nothing in int fields for products (events, banquet tickets, lunch tickets etc. NOT electrical outlets, packages to/from fair). See if they are removed from database
3. Try to add product to the field you just removed and save. See if they appear correctly in database

I think that was most of it!
